substrate/utils/frame/merklized-distribution/README.md

```diff
@@ -0,0 +1,3 @@
+# Merklized Distribution
+
+TODO
```

Comment:
Can you please add a minimal example? We recently cleaned up all READMEs in the repo.